packages/board/package.json

```diff
@@ -9,10 +9,14 @@
     "dev": "yarn --silent && yarn run -T parcel"
   },
   "devDependencies": {
+    "@parcel/config-default": "^2.1.1",
+    "@parcel/transformer-typescript-tsc": "^2.1.1",
     "@types/react": "^17.0.38",
-    "@types/react-dom": "^17.0.11"
+    "@types/react-dom": "^17.0.11",
+    "typescript": ">=3.0.0"
   },
   "dependencies": {
+    "@botpress/messaging-base": "0.0.1",
```

Comment:
I think that with every local packages, we should use the `*` instead of a fixed version as we always want to use the one provided locally!

Reply:
Ok but we can make this change in another PR. This is not consistent across the codebase